src/feed/Feed.js

```diff
@@ -98,6 +101,12 @@ class Feed {
                 });
                 callback({ quotes: [] });
                 return;
+            } else {
+                response = await this._binaryApi.getTickHistory(dataRequest);
+                this._mainStore.notification.notify({
+                    text: t.translate(errorMessage),
```

Comment:
This is not necessary; we pass the language to backend. What comes out here should be translated